recipes/esys-escript/meta.yaml

```diff
@@ -0,0 +1,78 @@
+
+package:
+    name: esys-escript
+    version: 5.4
+source:
+    - url: https://launchpad.net/escript-finley/3.0+/5.4/+download/python-escript_5.4.tar.gz
+      sha256: 61e1b3b5451d1eca06ab7c3905adfdc41906ed69e8388bfc4db094b944eb403b
+      folder: escript
+build:
+    skip: True  # [osx]
+    skip: True  # [win]
+    skip: True  # [py!=27]
+    number: '0'
+    rpaths:
+        - ${PREFIX}/bin
+        - ${PREFIX}/include
+        - ${PREFIX}/lib
+        - ${SP_DIR}
+requirements:
+    build:
+        - {{ compiler('cxx') }}
+        - svn
+        - scons >3.0.5
+        - cmake
+        - make
+        - python 2.7.*
+        - openmp  # [linux]
+        - llvm-openmp  # [osx]
+    host:
+        - boost-cpp 
+        - boost
+        - python 
+        - suitesparse
+        - matplotlib
+        - gdal 
+        - pyproj 
+        - scipy 
+        - sympy
+        - openmp  # [linux]
+        - llvm-openmp  # [osx]
+    run:
+        - boost-cpp 
+        - boost
+        - numpy 
+        - python
+        - suitesparse
+        - matplotlib 
+        - gdal 
+        - pyproj
+        - scipy
+        - sympy
+        - openmp  # [linux]
```

Comment:
Same here

Reply:
Done

Reply:
Did you mean that I should remove `llvm-openmp`? I think that including `llvm-openmp` instead of `openmp` will lead to an error during compilation as the buildbot seems to be using the gcc compiler.

Reply:
No, it's fine now